install_config/install/advanced_install.adoc

```diff
@@ -426,6 +426,203 @@ openshift_examples_modify_imagestreams=true
 |Set to `true` if pointing to a registry other than the default. Modifies the image stream location to the value of `*oreg_url*`.
 |===
 
+[[advanced-install-configuring-integrated-glusterfs-backed-registry]]
+=== Configuring an Integrated GlusterFS Backed Registry
+
+An integrated GlusterFS Registry can be configured and deployed during the initial installation
+of the cluster to offer a more reliable image registry. link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html/container-native_storage_for_openshift_container_platform_3.4/[Red Hat Container Native Storage (CNS)] has been integrated into the {product-title} Advanced Installation procedure utilizing containerized GlusterFS storage pods to back the Docker registry.
+
+Some additional prerequisites and host preparation are required beyond the normal {product-title} xref:../../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Prerequisites]. 
+
+.Additional Prerequisites and Host Preparation
+[options="header"]
+|===
+
+|Prerequisite |Description
+|`*Storage Nodes*`
+|A minimum of 3 storage nodes must be available to run GlusterFS integrated registry (3-way replication)
+ 
+ Each storage node must have at least 1 raw block device with at least 10GB of free storage (2GB for the distributed heketi.db and 8GB for registry storage)
+
+|`*Enable Storage Repo*`
+|rh-gluster-3-for-rhel-7-server-rpms will need to be enabled for installation of *heketi-client*. 
+
+      `# subscription-manager repos --enable=rh-gluster-3-for-rhel-7-server-rpms`
+
+|`*Install Components*`
```

Comment:
Change this to `Required Software` or something like that. As mentioned above, the notion of RPMs doesn't apply to Atomic OSes. You should say something like "The following software/components are required", and them mention the programs directly. In the case of `heketi-cli`, point to the GitHub repo: https://github.com/heketi/heketi/releases .